compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/optimizer/LargeMethodOptimizer.java

```diff
@@ -75,13 +87,17 @@ public class LargeMethodOptimizer {
     private static final Name DEFAULT_WORKER_NAME = new Name("function");
     private static final String OBJECT_INITIALIZATION_FUNCTION_NAME = "$init$";
     private static final String SPLIT_METHOD = "$split$method$_";
+    private static final String ARG_INDEX = "%argIndex";
+    private static final String ARG_TUPLE_SIZE = "%argTupleSize";
+    private static final String ARG_TUPLE_TYPEDESC = "%argTupleTypeDesc_";
+    private static final String ARG_TUPLE = "%argTuple_";
     private final SymbolTable symbolTable;
     // splits are done only if the original function has more instructions than the below number
     private static final int FUNCTION_INSTRUCTION_COUNT_THRESHOLD = 1000;
     // splits are done only if the newly created method will contain more instructions than the below number
     private static final int SPLIT_INSTRUCTION_COUNT_THRESHOLD = 25;
     // splits are done only if the newly created method will have less function arguments than the below number
-    private static final int MAX_SPLIT_FUNCTION_ARG_COUNT = 125;
+    private static final int MAX_SPLIT_FUNCTION_ARG_COUNT = 93;
```

Comment:
Shall we make this consistant number like 100 or 80

Reply:
Test are failing when I change this to 100. Changed this to 90.